server/routes/api.js

```diff
@@ -39,11 +39,13 @@ export default (storage) => {
   }));
 
   const auth0 = middlewares.managementApiClient({
-    domain: config('AUTH0_DOMAIN')
+    domain: config('AUTH0_DOMAIN'),
+    clientId: config('AUTH0_CLIENT_ID'),
```

Comment:
Why did this work without a clientID/clientSecret? It shouldn't had: https://github.com/auth0-extensions/auth0-extension-tools/blob/c8d9b88e37c3174cf4eaeb7bf7e3c3c0f20550bf/src/auth0/managementApi.js#L72

Reply:
This is using the express middleware that uses the token issued to the user to call API2 so there is no need to have a client ID https://github.com/auth0-extensions/auth0-extension-express-tools/blob/master/src/middlewares/managementApiClient.js#L7-L11

Reply:
Thanks, missed the access token was being resolved from the `req.user` 🙇 .